Sources/StreamChatSwiftUI/ChatChannel/Reactions/ReactionsOverlayView.swift

```diff
@@ -62,6 +62,14 @@ public struct ReactionsOverlayView<Factory: ViewFactory>: View {
                     Alert.defaultErrorAlert
                 }
             
+            if !messageDisplayInfo.message.isSentByCurrentUser {
+                factory.makeMessageAvatarView(for: messageDisplayInfo.message.author)
+                    .offset(
+                        x: padding / 2,
+                        y: originY + messageDisplayInfo.frame.height - padding
```

Comment:
There is the slightest of jumps here. I played aroung and noticed when I add the (very random) value of + 2 here, the avatar icon stays perfectly in place.
Not sure it will introduce other regressions, so feel free to discard this.

```suggestion
                        y: originY + messageDisplayInfo.frame.height - padding + 2
```